opam

```diff
@@ -34,9 +34,11 @@ remove: [
 depends: [
   "ocamlfind" {build}
   "re"
-  "sexplib" {>="109.53.00"}
+  "sexplib" {>= "113.24.00"}
```

Comment:
Ah! Great! I did not realize that there was overlap between ppx and camlp4 sexplib versions! I will relax the constraint back to where it was.